src/components/FileUpload.js

```diff
@@ -6,32 +6,66 @@ import FileDropzone from "./FileDropzone";
 
 import "../styles/FileUpload.css";
 import "../styles/Link.css";
+import "../styles/error.css";
+
+import { useNavigate } from "react-router-dom";
+import toast, { Toaster } from 'react-hot-toast';
 
-import { Link } from "react-router-dom";
 export default function FileUpload({ setFileId }) {
-  const [pdfFile, setPdf] = useState(null);
+  const [pdfFile, setPdf] = useState(null); // Initialize with a value !=NULL pdfFile to render correctly the first time. Can be set to null after the user dop the file
```

Comment:
Please no comments in source code

Reply:
Sorry that was supposed to be only for me then I forgot it there

Reply:
No worries, just commit everything without comments